trieste/models/interfaces.py

```diff
@@ -380,7 +408,7 @@ def get_trajectory(self) -> TrajectoryFunction:
         raise NotImplementedError
 
 
-class FastUpdateModel(ABC):
+class FastUpdateModel(ProbabilisticModel):
```

Comment:
Was this change required in the end? If not might make sense to leave for now and address later?